packages/vite/src/node/server/index.ts

```diff
@@ -616,7 +617,11 @@ async function startServer(
 
       printServerUrls(hostname, protocol, port, base, info)
       // print shortcuts info
-      info(`  > Shortcuts: "r" restart, "o" open browser, "f" force restart`)
+      info(
+        `  > Shortcuts: ${SHORTCUTS.map(
+          (item) => `${item.name} ${item.desc}`
```

Comment:
Maybe we should use quotes here

```suggestion
          (item) => `"${item.name}" ${item.desc}`
```